guidance/library/_json.py

```diff
@@ -1,10 +1,19 @@
-from typing import Any, Callable, Mapping, MutableMapping, Optional, Sequence
+from typing import Any, Callable, Mapping, MutableMapping, Optional, Sequence, Union
+from json import dumps as json_dumps
 
 import guidance
 from guidance.library import char_range, one_or_more, optional, zero_or_more
 
 from .._grammar import GrammarFunction, select
 
+def to_compact_json(target: Any) -> str:
```

Comment:
I'm going back and forth on whether this should be `_to_compact_json()`, since it isn't really something we want to export from the library (except for use by the tests).

Reply:
I'll make it private :)